assets/javascript/app.js

```diff
@@ -0,0 +1,225 @@
+        text: "Balerion The Dread"
+        ,
+        text: "Aegon I Targaryen"
+        ,
+        text: "Aerys II Targaryen/The Mad King"
+        ,
+        text: "Tywin Lannister"
+    },
+    correct: "Balerion The Dread"
+}, {
+    text: "8. Which of the following is not one of Jamie Lannister's alias",
+    option: {
+        text: "Kingslayer"
+        ,
+        text: "Kingkiller"
+
+        ,
+        text: "Lord of Casterly Rock"
+        ,
+        text: "Lord Commander of the Kingsguard"
+    },
+    correct: "Lord of Casterly Rock"
+}, {
+    text: "9. Which of the following houses extinct?",
+    option: {
+        text: "Lannisters"
+        ,
+        text: "Tyrells"
+        ,
+        text: "Targaryens"
+        ,
+        text: "Baratheons"
+    },
+    correct: "Tyrells"
+}, {
+    text: "10. Who is currently The King/Queen of the Seven Kingdom?",
+    option: {
+        text: "Bran Stark"
+        ,
+        text: "Jon Snow"
+        ,
+        text: "Daenerys Targaryen"
+        ,
+        text: "No One"
+    },
+    correct: "No One"
+}]
+
+// start
+function start() {
+    time = 120;
+    var currentTime = timeConverter(time);
+    $timer.text(currentTime);
+    intervalID = setInterval(countDown, 1000);
+}
+function countDown() {
+    time--;
+    var currentTime = timeConverter(time);
+    $timer.text(currentTime);
+    if (time === 0) {
+        endGame();
+    }
+}
+
+    
+
+
+function buildQuestions() {
+    for (var i = 0; i < questions.length; i++) {
+        var $question = buildQuestion(questions[i], i);
+        $trivia.append($question);
+    }
+    console.log($question);
+    var $button = $('<button type="submit" class="btn btn-primary">Submit Answers</button>');
+    $trivia.append($button);
+}
+
+function buildQuestion(question, index) {
+    var $fullQuestion = $('<div class="full-question">');
+    var $question = $('<h1 class="question">');
+    $question.text(question.text);
+    $fullQuestion.append($question);
+    var len = question.responses.length;
+    for(var i = 0; i < len; i++) {
+        var response = question.responses[i];
+        var $response = buildResponse(response, index);
+        $fullQuestion.append($response);
+    }
+    console.log($fullQuestion);
+    return $fullQuestion;
+}
+
+function buildResponse(response, index) {
+    var $responseInput = $('<input class="form-check-input" type="radio">');
+     $responseInput.addClass(questionName);
+    var $responseLabel = $('<label class="form-check-label">');
+    $responseLabel.text(response.text)
+    var $input = $('<div class="form-check">');
+    $input.append($responseInput);
+    $input.append($responseLabel);
+    return $input;
+}
+
+
+
+
+function endGame() {
+    //TODO:
+    //Show the score and right answer and wrong answer
+    if($input===option){
+        console.log("win");
+        right++;
+    } else {
+        wrong++;
+    }
+}
+start();
+buildQuestions();
+// the player will answer 10 multiples questions
```

Comment:
you call the first function buildquestions but buildquestion is not being called. That could maybe fix the issue with no questions appearing.